src/Humanizer/MetricNumeralExtensions.cs

```diff
@@ -1,8 +1,8 @@
-﻿// Wrote by Alois de Gouvello https://github.com/aloisdg
+﻿// Written by Alois de Gouvello https://github.com/aloisdg with additions by Jonas Barkå https://github.com/JonasBarka.
```

Comment:
The copyrights at the top of the file, if any, should be simply `Copyright (c) .NET Foundation and Contributors`

Alternatively, it can be removed as it's in the project root.